### ISSUE TITLE
AppImage does not start

### ISSUE THREAD

**Issue by srevinsaju** [SwagLyrics/SwagLyricsGUI#8]
On Xubuntu 18.04
![image](https://user-images.githubusercontent.com/48695438/89317694-6c59af00-d686-11ea-8850-40db216bb24f.png)

It looks like you are trying to install `swaglyrics` on spot using `pip`. On Xubuntu 18.04, Ubuntu 18.04 and above, `python` does not include a `pip` module by default. 

AppImage tests failed with this error (I am not sure what it means, related to C#)
```
===========================================
============= TRYING TO RUN ===============
===========================================
/tmp/.mount_BeingTsTRsEG/AppRun: POSIX shell script, ASCII text executable
-rwxr-xr-x 1 root root 206 Jul 20 14:14 /tmp/.mount_BeingTsTRsEG/AppRun
kernel.unprivileged_userns_clone = 1
Failed to load �]�, error: /var/tmp/.net/SwagLyricsGUI/bvctffl2.1wn/libhostfxr.so: failed to map segment from shared object
The library libhostfxr.so was found, but loading it from /var/tmp/.net/SwagLyricsGUI/bvctffl2.1wn/libhostfxr.so failed
  - Installing .NET Core prerequisites might help resolve this problem.
     https://go.microsoft.com/fwlink/?linkid=2063370
NUMBER_OF_WINDOWS: 0
ERROR: Could not find a single window on screen :-(
```

Possibly the bundle `libhostfxr.so` was not bundled on the AppImage, adding that library should fix the error. 

PS: Ideally, please package `swaglyrics` and `python3` into the appimage. Sandboxed appimages will not be able to run `/usr/bin/python`

**Comment by probonopd** [comment]
Also you might want to check out https://github.com/AppImage/awesome-appimage#deployment-tools-for-python-applications

**Comment by flabbet** [comment]
Yeah, makes sense. I guess we should do that. A good solution would be to create distro packages for major distributions, but it's a lot of work too. I don't have the time recently, so if you guys want you can make a PR with lib freeze solution, feel free to do so.
